src/cargopacket.h

```diff
@@ -538,8 +538,8 @@ class StationCargoList : public CargoList<StationCargoList, StationCargoPacketMa
 	uint Reroute(uint max_move, StationCargoList *dest, StationID avoid, StationID avoid2, const GoodsEntry *ge);
 
 	/**
-	 * Are two the two CargoPackets mergeable in the context of
-	 * a list of CargoPackets for a Vehicle?
+	 * Are the two CargoPackets mergeable in the context of
+	 * a list of CargoPackets for a Station?
```

Comment:
Is this change right? Without looking at the context, I would say that "days_in_transit" would be for a vehicle, not a station

Reply:
The comment had been copy/pasted verbatim from the function of the same name for `Vehicle`. To me, the two functions look to be called in different contexts; however, it would certainly be worth looking into.

Reply:
I would assume that, for a cargo packet in a station, `days_in_transit` would be the time elapsed since the cargo left its station of origin. Perhaps someone familiar with cargodist can verify this?

Reply:
`days_in_transit` is the number of days the cargo has been on board a vehicle. Time spent sitting on a station does not count.
Whether the list of CargoPackets is for a vehicle or station should be irrelevant, since they can potentially be merged both during loading and unloading. (As far as I know.)

Reply:
I think the question here is the context in which `AreMergable` is called. By my reckoning, one is called in the context of a cargo packet being added to a vehicle, and the other is called in the context of a cargo packet being added to a station.

I could be wrong, of course; however, that is my interpretation based upon a day or two of studying the code.

Reply:
I have split the problematic line into a separate commit. I will file a separate pull request for it.